lib/rubocop/cop/mixin/target_ruby_version.rb

```diff
@@ -8,12 +8,23 @@ def required_minimum_ruby_version
         @minimum_target_ruby_version
       end
 
+      def required_maximum_ruby_version
+        @maximum_target_ruby_version
+      end
+
       def minimum_target_ruby_version(version)
         @minimum_target_ruby_version = version
       end
 
+      def maximum_target_ruby_version(version)
+        @maximum_target_ruby_version = version
+      end
+
       def support_target_ruby_version?(version)
-        required_minimum_ruby_version <= version
+        min = required_minimum_ruby_version || -Float::INFINITY
```

Comment:
I think we need to have some comment here explaining this bit of code as it's not super obvious what we're aiming for - no matter if it's using 0 or something else.

Reply:
Makes sense, I added a comment, please let me know if it's clear.